### DISCUSSION TITLE
Phare Fantôme

### DISCUSSION THREAD

**Discussion by ghost** [Crownicles/Crownicles#1978] [Suggestions fermées]
> Nouvelle zone se décomposant en 3 Évents : Devant le phare, La Mer Azuréenne (nom à débattre si vous voulez) et le vieux port.
> Pour toute question : Ntalcme#3858 
> Vous pouvez aussi demander le fichier PDF de la zone complète.

Devant le Phare

Vous marchez paisiblement sur le sable, en direction de la mer, quand soudainement vous apercevez ce bâtiment si mystérieux : le Phare Fantôme. Terrifiant, lugubre et délabré sont les mots qui lui sont attribués. Battu par les vents soufflants de la mer, celui-ci menace de s'écrouler mais étrangement rien ne vient à bout de ce vieux bâtiment. Pris d'aventure et de curiosité, vous hésitez à explorer cet endroit qui respire le danger mais votre prudence vous met en garde et vous incite à ne pas courir le risque.

🚪: Visiter le rez-de-chaussée
↗️ : Monter au sommet du phare
🔎 : Fouiller les alentours
🔚 : Ne pas répondre

———————————————

🚪Visiter le rez-de-chaussée
Vous entrez prudemment dans l'édifice, il y fait sombre mais surtout si froid. Vous avancez prudemment, éclairé par la lumière extérieure dans ce bâtiment étonnamment grand quand soudain, le vent souffle d'une puissance gigantesque et l'entrée du phare se ferme brusquement, vous plongeant dans la pénombre totale. Perdu dans cet immense phare, vous êtes immobilisé par le froid avant de réussir à vous en sortir.
🥶 1h | -5 💔 

Vous pénétrez lentement dans le bâtiment, il y fait sombre. Vous avancez discrètement vers ce que vous croyez être une lanterne afin de vous éclairer mais votre pied bute sur un objet énorme. Il est mou et poilu, ce n'est rien d'autre qu'un gros rat qui, pris de colère, vous pourchasse. Tout d'un coup, des dizaines de ses congénères sortent de nul part afin de le défendre. Assailli de tout part, vous prenez tant bien que mal la fuite.
🤕 6h | -25 💔

Une fois à l'intérieur du phare, vous arrivez à distinguer tant bien que mal une lanterne. Vous vous dirigez vers celle-ci afin de l'allumer et aussitôt cela fait, vous apercevez une malle. Vous vous approchez vers elle dans le but de l'ouvrir mais celle-ci était fermée alors vous tentez de l'ouvrir plusieurs fois en lui donnant des coups mais ce fut vain. Vous repartez bredouille.
🕑 15m | +25 ⭐

Une fois entré dans le phare, vous arrivez à voir ce qui semble être une lanterne alors vous vous dirigez vers elle et l'allumez. Aussitôt la pièce éclairée, vous apercevez une malle dans un recoin. Pris de curiosité, vous avancez vers elle et tentez de l'ouvrir. Après plusieurs minutes, vous parvenez à vaincre la serrure.
+250 💰 | 🕑10m | +125 ⭐

Vous entrez dans le bâtiment. Le rez-de-chaussée était éclairé d'une faible lueur provenant d'une lanterne, de plus vous découvrez sur la table un reste de repas, visiblement un voyageur était passé par là il y a peu de temps. Vous profitez donc de ce faible festin.
+5 ❤️ | 🕑 10m | +225 ⭐

—————

↗️ Monter au sommet du phare
Vous entrez rapidement dans le phare et entamez une grimpée vers son sommet, pressé de découvrir la vue qu'il a à vous offrir. Celle-ci était magnifique, le Soleil à son zénith offrait de jolis reflets sur la mer bleue. Vous admirez ce splendide spectacle si longtemps que vous en perdez la notion du temps. Vous repartez tout reposé par la beauté de la nature.
🕑 2h | +10 ❤️ | +225 ⭐

Aussitôt entré dans le phare, vous commencez à monter ses escaliers afin d'atteindre le sommet. Une fois arrivé en haut, vous tombez nez-à-nez avec le vieux mécanisme du phare, vous en êtes tout émerveillé. Pris d'un élan passionné de voir à nouveau le phare s'éclairer, vous tentez de réparer le système. Un mystérieux blocage empêchait le vaste miroir de tourner alors vous inspectez le système d'engrenages et découvrez un objet. Après l'avoir enlevé et rangé dans votre inventaire, le miroir s'était remis à tourner mais malheureusement, aveuglé par votre désir de voir le phare allumé, vous aviez oublié que celui-ci n'avait plus de quoi éclairer et vous repartez triste.
🕑 3h | + item | +325 ⭐

Vous pénétrez avec hâte dans le phare et vous courez vers les escaliers tout excité de voir la vue à son sommet mais vous semblez oublier que le phare est ancien et à l'abandon depuis un certain temps, les escaliers en bois n'étaient pas très solides et dans votre précipitation l'un s'affaissa sous votre pied. Vous tombez violemment à travers.
😖 40m | -25 💔 

—————

🔎 Fouiller les alentours 
À la place de pénétrer le sinistre bâtiment, vous décidez plutôt d'explorer ses alentours dans l'optique de dénicher quelque chose qui pourrait vous être utile. Après quelques minutes de marche, vous ne trouvez rien d'autre qu'un petit pommier solitaire dont les fruits ont l'air appétissants. Vous décidez donc de cueillir un de ceux-ci et de manger doucement. C'était succulent ! La pomme était d'un goût unique que vous n'aviez jamais connu auparavant. Après avoir profité de cette petite collation, vous reprenez votre chemin.
+5 ❤️ | +350 ⭐

En vous baladant aux alentours du phare, vous découvrez une cache derrière un tas de buissons. Mais vous êtes bien trop imposant et, de peur, un serpent vous fonça dessus avant de s'enfuir. Bien que choqué, vous n'êtes pas blessé alors vous fouillez cette cache. Elle ne vous offre qu'un maigre butin. Vous repartez après l'avoir collecté.
+50 💰 | 😱 10m 

Vous faites un rapide tour des environs mais ne voyant rien digne d'intérêt, vous repartez aussitôt.
🕑 5m | +25 ⭐

—————

🔚 Ne pas répondre 
Hésitant, vous entrez tout de même dans le phare, il y fait sombre mais vous arrivez à distinguer une lanterne alors vous vous dirigez vers celle-ci afin de l'allumer. Aussitôt cela fait, vous voyez un livre sur une table et celui-ci attisa votre curiosité. Une fois la main mise dessus, vous l'ouvrez, il était très poussiéreux. Soufflant dessus, vous lisez les premières lignes « Il était une fois un vieux pêcheur. Champion dans son domaine, il avait pêché toutes les espèces du monde : gros et petits, colorés ou non, des poissons plus insolites les uns que les autres. Mais un jour, durant un de ses périples sur la mer bleue, il pêcha comme à son habitude quand soudainement une immense ombre apparût sous l'eau et voilà que… ». La suite était illisible, vous repartez perplexe et curieux de savoir ce que pouvait être la suite.
🕑 10m | +25 ⭐

Admirant le phare, plein de curiosité, un flot de questions vous inonde :  "Depuis combien de temps est-il maintenant debout, quelles histoires a-t-il bien pu vivre ? Comment était-il ?" Après quelques instants de réflexion, vous repartez tout en vous imaginant de nombreuses choses à son sujet.
+150 ⭐

Vous hésitez tellement longuement entre explorer le phare ou continuer votre route qu'un voleur en profite pour se faufiler discrètement derrière vous afin de vous dépouiller. Les yeux rivés sur le phare, vous ne vous en rendez même pas compte et c'est seulement après avoir quitté votre esprit que vous constatez la situation. Vous repartez, agacé par votre inattention. 
200 💸 | +150 ⭐

La Mer Azuréenne 

Vous vous tenez maintenant face à la majestueuse Mer Azuréenne, un océan tumultueux mais d'une beauté inégalée. Les vagues virevoltent sous l'éclatant Soleil qui brûle les yeux. Selon la légende ancestrale, un redoutable monstre marin se tapit dans les abysses de ces eaux mystérieuses. Votre âme d'aventurier frémit d'excitation à l'idée d'embarquer et de partir à la recherche de cette créature, prête à percer les secrets les plus profonds de la mer. Vous voyez une petite embarcation à voiles, prudence et sûreté contre curiosité et aventure se mêlent alors dans votre esprit et vous réfléchissez calmement, vous pensez alors prendre la mer…

🛶 Prendre la mer
❌ Oublier l'idée
🚶 Se balader sur la digue
🔚 Ne pas répondre
———————————————
🛶 Prendre la mer 
Vous décidez de prendre la mer et vous entamez votre croisière. Après une longue navigation, vous apercevez une île au loin. Vous décidez donc de vous dirigez vers celle-ci quand tout d'un coup vous entendez un rugissement terrifiant que vous en êtes pétrifié. Aussitôt votre barque détruite, vous vous retrouvez dans l'eau, une ombre gigantesque surgit alors des profondeurs ! Vous n'avez même pas le temps de distinguer la bête que celle-ci vous assène un coup de queue. Vous mourrez instantanément.
☠️ Mort

Vous vous lancez sur les vagues de la mer avec prudence, après quelques temps de navigation vous distinguez une ombre sous l'eau, c'est forcément le monstre marin ! Vous naviguez prudemment afin de venir à bout de cette bête mais là est votre bêtise, les mouvements des vagues faites par votre embarcadère réveillent le monstre et celui-ci fonce sur vous ! Surpris par la rapidité de celui-ci, vous vous préparez au combat tant bien que mal et par vous ne savez quel miracle vous lui assénez un coup fulgurant qui le fait fuir. Vous n'en ressortez tout de même pas indemne et rejoignez vite la côte.
😵 12h | 100 💔

Vous prenez la mer plein de courage, bien décidé à dénicher ce monstre. Après une longue navigation, vous voyez un rocher sortant de l'eau, intrigué vous vous dirigez vers celui-ci, il était à peine plus grand que votre embarcation et un cadavre gisait sur celui-ci. Un peu pris de peine par le spectacle devant vous, vous décidez tout de même de prendre les affaires du squelette que vous rangez dans les vôtres. Vous prenez un temps pour remercier le mort puis vous vous reposer avant de retourner sur la terre ferme.
+500 💰 | +item | +20 ❤️ | 🕑 3h | +825 ⭐

Vous tentez de pousser un peu plus le bateau dans l'eau que celui-ci cède, sûrement à cause de son entretien qui n'avait pas été fait depuis une longtemps. Vous reprenez donc votre voyage.
+150 ⭐

—————
❌ Oublier l'idée

Vous abandonnez l'idée et reprenez paisiblement votre chemin.
+150 ⭐

Vous vous approchez du bateau de près avant de renoncer mais un groupe d'aventuriers malintentionnés passant par là vous remarque et vous attaque. Afin d'éviter une humiliation, vous leur donnez votre argent.
350 💸 | +150 ⭐

—————
🚶Se balader sur la digue

Vous vous baladez sur la digue en profitant de la vue que la mer peut vous offrir. L'air est bon à respirer et le bruit des vagues agréable à écouter. Après une courte balade, vous reprenez votre chemin.
🕑 1h | +5 ❤️ | +225 ⭐

Vous vous promenez le long de la mer mais la chaleur insupportable ne fait que vous ralentir et vous en finissez immobile. Vous attendez une baisse de température avant de reprendre votre chemin.
🥵 1h

Vous marchez tranquillement au bord de l'eau quand soudain le vent monte. Une vague vient vous percuter d'une violence inouïe et vous assomme.
😴 3h | +20 ⭐

—————
🔚 Ne pas répondre
Vous prenez la mer malgré le temps venteux. À peine quelques minutes après votre départ, le vent vous fait perdre le contrôle de votre bateau et vous foncez sur des récifs. Une douleur extrême se fait ressentir.
80 💔 | 🤕 6h

À la place de prendre le bateau pour traquer le monstre, vous décidez de passer du bon temps sur le bateau, tel une croisière. Vous vous assoupissez et laissez emporter au gré de la mer. Après un bon repos, vous remarquez que votre bateau s'est amarré au bord d'une grotte. Vous décidez donc de l'explorer et y découvrez une cache au trésor. Après avoir tout récupéré, vous reprenez le bateau afin de retrouver votre emplacement initial. Tout ceci vous a pris la journée.
🕑 24h | +3500 💰 | +5 ❤️ | +325 ⭐

Le Vieux Port

Vous voilà désormais dans l'enceinte du vieux port. Bien qu'il soit ancien, il n'est pas complètement déserté et vous pouvez y trouver quelques commerces et habitations de pêcheurs. Vous marchez tranquillement quand vous surprenez une conversation entre deux hommes : « Le vieux capitaine proclame qu'il a vu une île au loin, qu'il est fou ! ». Intrigué par cette histoire, vous hésitez sur quoi faire.

⚓ Aller voir le vieux capitaine
👛 Aller chez un commerçant du coin
🛏️ Aller à l'auberge
🔚 Ne pas répondre

⚓ Aller voir le vieux capitaine 
Vous allez à la rencontre du vieux capitaine et lui demandez plus d'informations à propos de cette île. Après quelques échanges, vous décidez que celui-ci vous emmène sur cette fameuse île. Aussitôt arrivé sur celle-ci, une tribu, coupée du monde, vous stoppe et vous enferme après coups et blessures. Après une longue détention, ceux-ci vous relâchent, comprenant que vous n'êtes pas une menace. Le capitaine et vous regagnez aussi vite votre bateau afin de rejoindre le continent.
🕑 18h | 75 💔

Vous rencontrez le vieux capitaine, il a des rides et de nombreuses cicatrices sur le visage. Vous écoutez son histoire mais vous n'êtes vraiment pas convaincu par cet homme loufoque. Vous décidez simplement de repartir en prétextant une excuse.
+150 ⭐

Vous rencontrez le vieux capitaine et écoutez son histoire. Ne croyant aucun mot de ce qu'il vous raconte, vous décidez quand même de profiter de la situation. En effet, vous lui proposez en échange d'une belle somme de l'accompagner vers l'île la nuit tombée. Une fois l'argent obtenu, vous quittez simplement le port.
+1000 💰 | +250 ⭐

—————

👛 Aller chez un commerçant du coin
Vous allez chez le forgeron du coin mais celui-ci n'avait pas grand-chose à vous proposer. Vous repartez aussitôt.
+150 ⭐

Vous allez chez le poissonnier et achetez un poisson. Vous vous posez plus loin sur un banc et mangez celui-ci. Il est si immonde que vous en avez mal au ventre.
🤢 6h | 10 💔

Vous partez acheter du poisson chez le poissonnier. Celui-ci vous dit qu'il a été pêché ce matin et qu'il est d'un goût excellent, vous en prenez donc un et le dégustez. Le poissonnier ne vous a pas menti, le poisson est succulent.
+350 ⭐ | 5 ❤️

Vous allez voir le marchand du coin, il vend toutes sortes d'objets. Après avoir fait le tour des objets, rien ne vous intéresse et vous êtes sur le départ. Voyant cela, le marchand vient vous voir et vous propose un objet unique qu'il a en stock et qui peut vous intéresser. Intrigué, vous achetez l'objet puis repartez.
+Item | +450 ⭐ | 200 💸

Vous allez chez le forgeron du coin. Vous regardez rapidement les armes qu'il propose et une attire particulièrement votre attention mais elle est trop chère pour vous. Vous attendez que le forgeron ne vous regarde plus avant de voler l'arme. Vous repartez le plus discrètement possible mais aussitôt le forgeron crie au voleur. La garde locale vous intercepte et vous fait prisonnier.
🔒24h 

—————

🛏️ Aller à l'auberge
Vous partez pour l'auberge comme la nuit tombe. Elle a l'air miteuse mais vous rentrez tout de même à l'intérieur. Le personnel est vraiment accueillant et vous installe directement à une table et vous réserve une chambre, il semble n'y avoir personne dans l'auberge. Après avoir mangé, vous vous endormez soudainement. En réalité, la nourriture contient un somnifère puissant et vous êtes dépouillé de vos affaires avant d'être jeté dehors.
😴 3h | 500 💸 | +20 ⭐

Vous allez vers l'auberge afin de prendre un bon repas. Une fois arrivé devant, son air miteux vous repousse mais vous entrez tout de même. Le personnel est chaleureux et vous êtes installé au milieu d'un brouhaha. Une fois votre plat arrivé, vous le dégustez et discutez avec les autres convives. Vous passez un agréable moment avant de repartir. 
+15 ❤️ | 🕑 2h | +225 ⭐

Vous prenez la direction de l'auberge afin de vous reposer mais une fois devant, son air miteux et délabré vous repousse. Vous repartez immédiatement.
+150 ⭐

—————

🔚 Ne pas répondre
Finalement, vous décidez d'interpeller les deux hommes afin de demander des renseignements sur les activités locales. Visiblement, pour pas cher, vous pouvez faire de la voile. Fatigué de votre voyage, vous voulez vous détendre, vous décidez donc d'en faire avant de repartir.
💸 50 | 🕑 1h | +25 ⭐

Finalement, non intéressé par cette histoire, vous choisissez plutôt de visiter le port. Vous pouvez facilement constater son ancienneté. Les seuls bâteaux amarrés appartiennent à des pêcheurs et le tout respiré la vieillesse. Vous repartez, non déçu de cette visite.
🕑 30m | +25 ⭐

**Comment by ghost** [comment]
![Screenshot_20230609_190809_ibisPaint X](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/assets/134647135/3c0407d3-1598-4943-b55b-b5394a878d85)

J'ai oublié l'image.

**Reply by BastLast** [reply]
Les events c'est pas ici
